src/fragments/lib/flutter.mdx

```diff
@@ -2,10 +2,7 @@
 
 Welcome to the Amplify Flutter documentation. To stay up to date with the latest changes and provide feedback, please take a look at our [Github repo](https://github.com/aws-amplify/amplify-flutter) or join us on [Discord](https://discord.gg/jWVbPfC).
 
-The stable release of Amplify Flutter currently supports iOS/Android as target platforms.
-
-This guide shows how to build an app using our Amplify Libraries for Flutter and the Amplify CLI toolchain.
```

Comment:
Do we want to keep this?

Reply:
I didn't remove it! weird that it is showing up that way. Can you check one more time @dnys1 ?